ox-manuscript/ox-manuscript.el

```diff
@@ -427,9 +427,9 @@ intermediate output steps."
       (goto-char (point-min))
       (setq run-makeglossary-p (re-search-forward "\\\\makeglossaries" nil t))
       (goto-char (point-min))
-      (setq run-bibtex-p (re-search-forward "bibliography:" nil t))
```

Comment:
you should not remove the : here or in the next one. I guess this is because you modified a file that was not current, the addition of : was pretty recent.